src/shared/components/DurationSelector.tsx

```diff
@@ -19,6 +28,8 @@ interface Props {
   disabled?: boolean
 }
 
+const durationUnits = ['d', 'mo', 'y']
```

Comment:
Currently Supported duration units. You can reasonably use these to create almost all required retention units. 
Example: 
`2 days`
`35 days`
`18 months`
`3 years`